lib/kb_Metrics/metricsDBs.py

```diff
@@ -232,9 +237,9 @@ def get_user_info(self, userIds, minTime, maxTime, exclude_kbstaff=False):
 
         signupTimeFilter = {}
         if minTime is not None:
-            signupTimeFilter['$gte'] = _convert_to_datetime(minTime)
+            signupTimeFilter['$gte'] = minTime #_convert_to_datetime(minTime)
```

Comment:
Could you verify source DB has int value instead of date? Either this will give wrong data or previous runs give wrong data.

Reply:
I know this one for sure.  I created this db according to the fields fetched from auth2, which saves dates in isodate.